src/components/donation/Steps/Submit/common/SplitSummary.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactNode } from "react";
+
+type Props = {
+  total: ReactNode;
+  locked: ReactNode;
+  liquid: ReactNode;
+  classes?: string;
+};
+
+export default function SplitSummary(props: Props) {
```

Comment:
Suggestion: related to the Split component, it would make sense to turn the cursor into pointer when hovering over the slider:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/cae9eacc-4acd-4cff-8f46-fa05e499c9aa)

Reply:
@ap-justin pinging you in case you missed this